api_app/helpers.py

```diff
@@ -130,3 +133,43 @@ def get_hash_type(hash_value):
             detected_hash_type = hash_type
             break
     return detected_hash_type  # stays None if no matches
+
+
+def get_custom_config_as_dict(user, entity_type, name=None) -> dict:
+    """
+    Returns custom config as dict
+    """
+    result = {}
+    try:
```

Comment:
while your intentions are clear to me, it would be useful to add more context (comments) to help the reader to understand why you check the org first

Reply:
please add also logs in critical parts of this function

Reply:
Done